pontoon/base/views.py

```diff
@@ -71,172 +55,49 @@ def home(request):
     return translate(request, locale, project.slug, path)
 
 
-def locale(request, locale):
-    """Locale view."""
-    l = get_object_or_404(Locale, code__iexact=locale)
-
-    projects = (
-        Project.objects.available()
-        .order_by('name')
-    )
-
-    if not projects:
-        raise Http404
-
-    return render(request, 'locale.html', {
-        'projects': projects,
-        'locale': l,
+def error403(request):
```

Comment:
Why do we need a separate view for 403? Django automatically loads 403.html when you throw authorization error?